pkg/sendgrid/sendgridapi.go

```diff
@@ -136,6 +137,24 @@ func (c *BackendAPIClient) DeleteSubUser(username string) error {
 	return nil
 }
 
+//DeleteAPIKey Delete api key of user with supplied username
+func (c *BackendAPIClient) DeleteAPIKey(keyID, keyName string) error {
```

Comment:
Let's maintain the naming convention here of `DeleteAPIKeyForSubUser`.

Currently we don't plan on deleting API keys for the master user, but it'll keep the API consistent in case we ever do

Reply:
:+1: